src/index.ts

```diff
@@ -1,6 +1,6 @@
 import type { Plugin } from 'vite';
 
-export default (): Plugin => {
+export function timeReporter(): Plugin {
```

Comment:
Why not using export default here?
But I'm okay with giving it a name, as this might be used by IDEs like VSCode for pre-variable-name auto-completion